application/helpers/update/updates/Update_628.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace LimeSurvey\Helpers\Update;
+
+use LimeSurvey\Helpers\Update\DatabaseUpdateBase;
+
+class Update_628 extends DatabaseUpdateBase
+{
+    /**
+     * @inheritDoc
+     */
+    public function up()
+    {
+        /* Delete old uneeded settings */
+        $this->db->createCommand(
+            "DELETE FROM {{settings_global}} WHERE " . $this->db->quoteColumnName('stg_name') . " LIKE 'last_survey_%'"
+        )->execute();
+        $this->db->createCommand(
+            "DELETE FROM {{settings_global}} WHERE " . $this->db->quoteColumnName('stg_name') . " LIKE 'last_question_%'"
+        )->execute();
+        $this->db->createCommand()->update('{{settings_global}}', array('stg_value' => 628), "stg_name='DBVersion'");
```

Comment:
Is this need?

Reply:
Yes, original issue : i have a config loaded each time with more than 6000 lines like this.

Reply:
And : **allways** remove undeeded part from DB are a good practice.

Reply:
I meant the setting the version... sorry

Reply:
Yes : else it don't happen .

There are no way to delete data one time only except update the database number.

Reply:
Not sure I follow. All other migration scripts don't do that, right?

Reply:
> Not sure I follow. All other migration scripts don't do that, right?

Arg, you're right. Misunderstanding the point; System do it automatically.

Reply:
https://github.com/LimeSurvey/LimeSurvey/pull/4182/commits/4433fa0b5c50e8dc811f2883c672a7616b208bd4